gammapy/scripts/image_pipe.py

```diff
@@ -101,6 +102,39 @@ def bkg_map(self, bkg_norm=True):
 
         self.maps["bkg"] = bkg_map
 
+    def make_1d_expected_counts(self, spectral_index=2.3, for_integral_flux=False):
+        """Compute the 1D exposure table for one observation for an offset table
+
+        Parameters
+        ----------
+        spectral_index : float
+            Assumed power-law spectral index
+        for_integral_flux : bool
+            True if you want that the total excess / exposure gives the integrated flux
+
+        Returns
+        -------
+        offset_tab: `numpy.array`
+            1D offset array in the FOV
+        exposure_tab: `numpy.array`
+            1D exposure array for the previous offset
+
+        """
+        energy = EnergyBounds.equal_log_spacing(self.energy_band[0].value, self.energy_band[1].value, 100,
+                                                self.energy_band.unit)
+        energy_band = energy.bands
+        energy_bin = energy.log_centers
+        eref = EnergyBounds(self.energy_band).log_centers
+        spectrum = (energy_bin / eref) ** (-spectral_index)
+        offset_tab = Angle(np.linspace(self.offset_band[0].value, self.offset_band[1].value, 10), self.offset_band.unit)
+        arf = self.aeff.evaluate(offset=offset_tab, energy=energy_bin).to("cm2").T
```

Comment:
Why? If we use quantities everywhere and propagate units, the flux will come out with some flux units.
It could be `m^-2` aor 'cm^-2' if you don't pin the output units.

But we should stop discussion this here ... that's a question we have to decide for 100s or functions in Gammapy and should discuss in a separate issue and then handle in a uniform way.

I just would prefer with starting more or less with a codebase in Gammapy where we always either:
1. don't pin units, output units depend on input units, or
2. pin units at the very end of the function, on the quantity that's returned.

If you do one of those two coding styles here it's fine with me. OK?